nixos/modules/services/misc/languagetool.nix

```diff
@@ -42,18 +45,29 @@ in {
           description = mdDoc "Number of sentences cached.";
         };
       };
-      default = {};
+      default = { };
       description = mdDoc ''
         Configuration file options for LanguageTool, see
         'languagetool-http-server --help'
         for supported settings.
       '';
     };
+
+    jrePackage = mkPackageOption pkgs "jre" { };
+
+    jvmOptions = mkOption {
+      description = lib.mdDoc "Extra command line options for the JVM running languagetool.";
```

Comment:
Can you add a link to the list of options in the doc string.

Reply:
Done